src/GithubPRPlanPlugin.groovy

```diff
@@ -179,7 +179,7 @@ class GithubPRPlanPlugin implements TerraformPlanCommandPlugin, TerraformEnviron
                 def output = sh(script: cmd, returnStdout: true).trim()
 
                 def headers = readFile('comment.headers').trim()
-                if (! headers.contains('HTTP/1.1 201 Created')) {
+                if (! (headers.contains('HTTP/1.1 201 Created') || headers.contains('HTTP/2 201 Created'))) {
```

Comment:
Do we really care about the HTTP version?  Could we simplify this to "201 Created"?

Not a change that you need to make, but arguably in the future, we could even git rid of the comment.headers, with a 

`[ curl -o output -I -L -s -w "%{http_code}" "${pullRequestUrl}" -eq 201 ]`

Reply:
I agree we can definitely simplify and check that. @jleopold28 thoughts?

Reply:
@bondezbond checking response code sounds good to me 👍

Reply:
done

Reply:
@kmanning during testing a colleague saw that http/2 only returns `201` now and drops `Created` part in response. So for now we'll have to to keep that complex conditional or we can move to a simplified curl call during this PR.